### DISCUSSION TITLE
What is default value of mock_outputs_allowed_terraform_commands when not provides?

### DISCUSSION THREAD

**Discussion by kamontat** [gruntwork-io/terragrunt#3599] [Q&A]
What is default value of mock_outputs_allowed_terraform_commands when not provides?

**Comment by yhakbar** [comment]
Hey @kamontat ,

By default it will mock all specified outputs when the dependency has not been applied.

If you could, would you mind submitting a PR to update the docs with a note you wish was there when you tried to look this up?
